src/functions/insert/handler.ts

```diff
@@ -0,0 +1,44 @@
+import createError from "http-errors";
+
+import type { ValidatedEventAPIGatewayProxyEvent } from "@libs/api-gateway";
+import { formatJSONResponse } from "@libs/api-gateway";
+import { middyfy } from "@libs/lambda";
+import { client } from "@services/dynamoDb";
+
+import schema from "./schema";
+
+const REASON_CODES = [...Array(11).keys()];
```

Comment:
clever, but reads poorly, could you wrap it around a generate range function and put it in utils?

Reply:
Certainly, will change it to:

```suggestion
const REASON_CODES = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
```